### DISCUSSION TITLE
Welcome to SMM1-Level-Downloader Discussions!

### DISCUSSION THREAD

**Discussion by HerobrineTV** [HerobrineTV/SMM1-Level-Downloader#86] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share Levels
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Kinglua243** [comment]
Are you supposed to download the entire thing or a certain folder?
Im confused
